src/model/repositories/AddressRepository.php

```diff
@@ -74,4 +73,25 @@ public function getAddressById(int $addressId): array {
       throw new Exception("Unable to fetch address");
     }
   }
+
+  /* Update address */
+public function updateAddress(int $addressId, string $street, string $streetNr, int $postalCodeId, int $postalCode, string $city): void {
+  try {
```

Comment:
How come we don't use a transaction here when we try to update two tables?

Reply:
Or two nested try blcoks?

Reply:
We don't use transaction, because a transaction has already been started before.
But! Very good point with the nested try blocks, since we could be more specific with the error message.